vmdb/app/models/ems_microsoft.rb

```diff
@@ -96,7 +96,8 @@ def execute_power_operation(cmdlet, vm_uid_ems, *parameters)
     return unless vm_uid_ems.guid?
 
     params  = parameters.join(" ")
-    command = "powershell #{cmdlet}-SCVirtualMachine -VM (Get-SCVirtualMachine -ID #{vm_uid_ems}) #{params}"
+    command = "powershell Import-Module VirtualMachineManager; Get-SCVMMServer localhost;\
```

Comment:
@supernoodz If it's feasible that `localhost` will ever change to IPv6 literal such as `::1`, I'd prefer we put a reminder of what @jerryk55 found regarding using `[::1]`: such as:

```ruby
# TODO: If localhost could feasibly be changed to an IPv6 address such as "::1", we need to wrap the IPv6 address in square brackets,  similar to the a URIs's host field, "[::1]".
command = "powershell Import-Module VirtualMachineManager; Get-SCVMMServer localhost;\
```